README.md

```diff
@@ -1 +1,7 @@
-# MicroverseProject1
\ No newline at end of file
+## Copyright 2023, [Isaac Gakure Wanyoike]
+
+
+
+This is a work done for Microverse projects.
+
+DONE ON 10/01/2023
```

Comment:
- [ ] As instructed by the previous reviewer, kindly make your README.md file look professional by using the recommended template provided by Microverse. Be sure to add the relevant headers that are necessary for this project. You can use the template link provided below to make yours:

https://github.com/microverseinc/readme-template

Reply:
Hello,  I cannot resubmit as it says it says the review is pending.

Reply:
@gaks1, you just resubmitted and I just claimed it again, so is there a reason why you stated this? 😄😆 👍

Reply:
> @gaks1, you just resubmitted and I just claimed it again, so is there a reason why you stated this? 😄😆 👍

@Gambit142 , sorry I did not reload the progress page so I was confused and panicked.

Reply:
@gaks1, it is alright. I understand. 👍